setup.py

```diff
@@ -46,8 +46,19 @@ def run(self):
     version=version,
     description="Super Resolving Renewable Resource Data (sup3r)",
     long_description=readme,
-    author="Ryan King",
-    author_email="ryan.king@nrel.gov",
+    author="Brandon Benton",
```

Comment:
Very generous with your credit attribution. I suppose this means I'll be getting the issue emails though. :)